Riot/Modules/SplitView/SplitViewCoordinator.swift

```diff
@@ -100,6 +102,10 @@ final class SplitViewCoordinator: NSObject, SplitViewCoordinatorType {
             
             // Setup split view controller
             self.splitViewController.viewControllers = [tabBarCoordinator.toPresentable(), detailNavigationController]
+            
+            // Setup detail user indicator presenter
+            let presentingViewController = splitViewController.isCollapsed ? tabBarCoordinator.toPresentable() : detailNavigationController
+            detailUserIndicatorPresenter = UserIndicatorTypePresenter(presentingViewController: presentingViewController)
```

Comment:
I wonder if we should pull this out into its own method and also call it from the 2 `UISplitViewControllerDelegate` methods too, to handle iPad layout changes.
```swift
SplitViewCoordinator.splitViewController(_:separateSecondaryFrom:)
SplitViewCoordinator.splitViewController(_:collapseSecondary:onto:)
```

Reply:
This makes sense, although the fix I just pushed is not sufficient yet either, because currently each time the layout is changed, the presenter will be recreated, incl creating a new indicator queue, which is bad. Because this is a relative edge case (ie you need iPad + detail shown + spinner active + change of layout) I'd go with this for now and in separete PR I want to hide the presentingViewController behind some kind of factory interface that gets passed into the presenter. That way only the view controller is changed on demand rather than the entire indicator presenter

Reply:
Makes sense to me 👍